src/__tests__/shared/components/navbar/Navbar.test.tsx

```diff
@@ -101,107 +103,80 @@ describe('Navbar', () => {
-
-      children.forEach((option: any) => {
-        expect(option.props.children).not.toEqual('labs.requests.new')
-        expect(option.props.children).not.toEqual('incidents.requests.new')
-        expect(option.props.children).not.toEqual('imagings.requests.new')
+        // 0 & 1 index are dashboard fixed elements, 2 index is first menu label for user
+        expect(screen.queryAllByRole('button')[2]).toHaveTextContent(/patients\.newpatient/i)
       })
     })
-  })
 
-  describe('account', () => {
-    it("should render a link with the user's identification", () => {
-      const expectedUserName = `user.login.currentlySignedInAs ${userName.givenName} ${userName.familyName}`
+    describe('account', () => {
+      it("should render a link with the user's identification", () => {
+        const { container } = setup(allPermissions, userName)
+        const navButton = container.querySelector('.nav-account')?.firstElementChild as Element
+        userEvent.click(navButton)
 
-      const wrapper = setup(allPermissions, userName)
-      const hospitalRunNavbar = wrapper.find(HospitalRunNavbar)
-      const accountLinkList = hospitalRunNavbar.find('.nav-account')
-      const { children } = accountLinkList.first().props() as any
+        expect(
+          screen.getByText(/user\.login\.currentlysignedinas givenname familyname/i),
+        ).toBeInTheDocument()
+      })
 
-      expect(children[0].props.children).toEqual([undefined, expectedUserName])
-    })
+      it('should render a setting link list', () => {
+        setup(allPermissions)
+        const { container } = setup(allPermissions, userName)
 
-    it('should render a setting link list', () => {
-      const wrapper = setup(allPermissions)
-      const hospitalRunNavbar = wrapper.find(HospitalRunNavbar)
-      const accountLinkList = hospitalRunNavbar.find('.nav-account')
-      const { children } = accountLinkList.first().props() as any
+        const navButton = container.querySelector('.nav-account')?.firstElementChild as Element
+        userEvent.click(navButton)
 
-      expect(children[1].props.children).toEqual([undefined, 'settings.label'])
-    })
+        expect(screen.getByText('settings.label')).toBeInTheDocument()
+      })
 
-    it('should navigate to /settings when the list option is selected', () => {
-      const wrapper = setup(allPermissions)
-      const hospitalRunNavbar = wrapper.find(HospitalRunNavbar)
-      const accountLinkList = hospitalRunNavbar.find('.nav-account')
-      const { children } = accountLinkList.first().props() as any
+      it('should navigate to /settings when the list option is selected', () => {
+        setup(allPermissions)
+        const { container } = setup(allPermissions, userName)
 
-      act(() => {
-        children[0].props.onClick()
-        children[1].props.onClick()
-      })
+        const navButton = container.querySelector('.nav-account')?.firstElementChild as Element
```

Comment:
there isn't an accessible way to get this nav button?

Reply:
I am aware. Like I said in chat this is how the test was convertible, we can't just ignore or remove their tests and if someone else can figure out another way to get this working later then all the better.